error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/util/MoreJUnitMatchers.java

```diff
@@ -0,0 +1,69 @@
+package tech.picnic.errorprone.bugpatterns.util;
+
+import static com.google.errorprone.matchers.ChildMultiMatcher.MatchType.AT_LEAST_ONE;
+import static com.google.errorprone.matchers.Matchers.allOf;
+import static com.google.errorprone.matchers.Matchers.annotations;
+import static com.google.errorprone.matchers.Matchers.anyOf;
+import static com.google.errorprone.matchers.Matchers.isType;
+import static tech.picnic.errorprone.bugpatterns.util.MoreMatchers.hasMetaAnnotation;
+
+import com.google.common.collect.Iterables;
+import com.google.errorprone.matchers.Matcher;
+import com.google.errorprone.matchers.MultiMatcher;
+import com.google.errorprone.util.ASTHelpers;
+import com.sun.source.tree.AnnotationTree;
+import com.sun.source.tree.AssignmentTree;
+import com.sun.source.tree.ExpressionTree;
+import com.sun.source.tree.MethodTree;
+import com.sun.tools.javac.code.Type;
+import java.util.Optional;
+import javax.lang.model.type.TypeKind;
+
+/** A set of JUnit-specific helpers for working with the AST. */
```

Comment:
Just for reference, the Javadoc from `JUnitMatchers` itself: 

>  * Matchers for code patterns which appear to be JUnit-based tests.

I think mentioning `AST` here isn't as accurate as we can be 😬. 

```suggestion
/** A set of JUnit Jupiter-specific helper methods and {@link Matcher Matchers}. */
```